pyproject.toml

```diff
@@ -1,17 +1,16 @@
 [build-system]
-requires = ["setuptools >=61", "wheel", "Cython >=3.0.5"]
+requires = ["setuptools >=77", "Cython >=3.0.5"]
```

Comment:
Are you sure `wheel` package is not needed? It is not listed in basic dependencies of `setuptools`, only in optional (extra)

Reply:
> Are you sure `wheel` package is not needed?

With `v70.1` setuptools started vendoring `wheel` and modifying it for its own use. Since then it has not relied on the `wheel` package itself.
https://setuptools.pypa.io/en/latest/history.html#v70-1-0

The official docs also only mention `setuptools` for `build-system.requires`.
https://setuptools.pypa.io/en/latest/build_meta.html#how-to-use-it

Furthermore, I doubled checked that `python -m build` still works.

Reply:
If we remove it from here, we have to remove it from requirements*.txt files too.

Reply:
Sorry, you're right, it was installed directly in CI.